### PR TITLE
Clean-Up : cleanup VirtualBox in packer build

### PR DESCRIPTION
**Pull request by panpan0000**
detail in [RAC-5045](https://rackhd.atlassian.net/browse/RAC-5045)

we saw failure in jenkins build, so we need to add this cleanup

### CODE REVIEW

**Comment by panpan0000** [comment]
@PengTian0

**Comment by panpan0000** [comment]
@johren  @anhou  @iceiilin 

Please help to review and this will fix blocking issue of Master CI

File: jobs/build_vagrant/cleanup_vbox.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        vboxmanage controlvm ${uuid} poweroff
+        echo "deleting vm ${uuid}"
+        vboxmanage unregistervm ${uuid}
+      done
+  set -e
+}
+# To resolve error message as below, which is a dirty worksspace
+# Build 'virtualbox-ovf' errored: Error enabling VRDP: VBoxManage error: VBoxManage: error: The machine 'rackhd-ubuntu-14.04' is already locked for a session (or being unlocked)
+
+virtualBoxDestroyAll
```
Comment (PengTian0) [inline jobs/build_vagrant/cleanup_vbox.sh]: We also need to remove the directory ```~/VirtualBox\\ VMs/rackhd-${OS_VER}```

**Comment by panpan0000** [comment]
per @johren , this PR will not be merged until Master restores.
I will do manual clean up for now.

File: jobs/build_vagrant/cleanup_vbox.sh (hunk 1 above)
Comment (panpan0000) [reply jobs/build_vagrant/cleanup_vbox.sh]: fixed. thanks

**Comment by PengTian0** [comment]
:+1:

File: jobs/build_vagrant/build_vagrant.sh [hunk 2]
```diff
@@ -4,13 +4,21 @@ ifconfig
 packer -v
 vagrant -v
 
+cleanup()
+{
+    BASEDIR=$(dirname "$0")
+    pkill packer
+    BASEDIR/cleanup_vbox.sh
```
Comment (changev) [inline jobs/build_vagrant/build_vagrant.sh]: $BASEDIR

Comment (panpan0000) [reply jobs/build_vagrant/build_vagrant.sh]: fixed. thanks

**Review by anhou — APPROVED** [review]

**Pull request merged**